library/src/main/java/com/novoda/downloadmanager/lib/BatchRepository.java

```diff
@@ -167,13 +170,11 @@ public DownloadBatch retrieveBatchFor(DownloadInfo download) {
     }
 
     public void deleteMarkedBatchesFor(Collection<DownloadInfo> downloads) {
-        Cursor batchesCursor = resolver.query(Downloads.Impl.BATCH_CONTENT_URI, null, SELECT_MARKED_FOR_DELETION, null, null);
+        Cursor batchesCursor = resolver.query(Downloads.Impl.BATCH_CONTENT_URI, PROJECT_BATCH_ID, SELECT_DELETED, WHERE_MARKED_FOR_DELETION, null);
```

Comment:
@friedger You are completely right, fixed all values and names https://github.com/novoda/download-manager/commit/86d0ef542b1e06684579a8e4201ba3428b942c5b

I prefer to use `MARKED_FOR_DELETION` instead of `DELETION_FLAG` as it might be more expressive to use the former and we don't have anywhere in the code right now for a `keep` value :)

Thanks for your eye!